### ISSUE TITLE
allow number format with leading zero

### ISSUE THREAD

**Issue by muescha** [FaisalUmair/udemy-downloader-gui#19]
what i do:
- i downloaded a section
- i open VLC 
- i select "Advanced File open"
- i selected the folder of a section

what i see:
- when there are lectures with >9 numbers 
- then the lecture 10 start right after lection 1

what i expect
- lectures in right order

how to fix:
- allow lecture and section numbers formatte with leading zero:
1 --> 001
2 --> 002
10 --> 010

**Comment by FaisalUmair** [comment]
Yeah I have seen this numbering scheme implemented in some of the scripts which download Udemy courses. I never understood why they use leading zeros but now I know. 

But if we generate a playlist per course and per section of course? Do you still need the leading zeros?

Thanks

**Comment by muescha** [comment]
i think some other apps (and the windows explorer ) not sort it right without a leeding zero. 
so it is a low hanging fruit to make an option "add leading zeros" to get vlc work ;)

**Comment by muescha** [comment]
or maybe on windows by default on.

for #16 you should be aware that someone changed this setting while download sessions.

**Comment by FaisalUmair** [comment]
I think you are right, better to implement it as it wont hurt as anyway.

**Comment by tofanelli** [comment]
i totally agree with this naming schema =)

**Comment by oscarreno** [comment]
Maybe this issue can be joined with

[(https://github.com/FaisalUmair/udemy-downloader-gui/issues/317)]

In fact, my comment there is in the same line of what is written here.

[https://github.com/FaisalUmair/udemy-downloader-gui/issues/317#issuecomment-690767290]
